PHPCS Cleanup

namespace-dave opened the pull request:
This PR resolves all PHPCS sniff violations and Fixes #83.

pramodjodhani commented:
I used it with Delivery Slots. I noticed 2 issues:

1. filemtime(): stat failed for app/public/wp-content/plugins/delivery-slots/inc/vendor/wp-settings-framework/assets/vendor/jquery-ui/jquery-ui.css in /app/public/wp-content/plugins/delivery-slots/inc/vendor/wp-settings-framework/wp-settings-framework.php on line 356

2. Icon in header:
<img width="1334" alt="image" src="https://user-images.githubusercontent.com/5794565/186880703-e2a99fe5-45eb-4fec-adc8-efbc6b2aec4a.png">

pramodjodhani reviewed (changes requested):
@namespace-dave Almost perfect. Please see my comment.

namespace-dave commented:
@pramodjodhani resolved both issues:

1. The files were actually missing from the repo due to the `vendor/` line in `.gitignore`. I've changed to `/vendor/` and committed to resolve.
2. I've switched to `wp_kses` along with passing in `data` as one of the allowed protocols, as WordPress does not allow `data` protocols in its escaping functions by default.

Tested with WDS and everything looks fine, merging!
